internal/xds/translator/testdata/in/xds-ir/tcp-route-authorization.yaml

```diff
@@ -0,0 +1,34 @@
+tcp:
+- name: "tcp-listener-authorization"
+  address: "::"
+  port: 10080
+  routes:
+  - name: "tcp-route-authorization"
+    authorization:
+      defaultAction: Allow
```

Comment:
Non-blocking nit: We can use Allow,  but Deny makes more sense here as an example.